SharedProjects/Babylon2GLTF/GLTFExporter.Material.cs

```diff
@@ -300,7 +303,14 @@ private void ExportMaterial(BabylonMaterial babylonMaterial, GLTF gltf)
                                 {
                                     logger.RaiseError("Diffuse, specular and opacity maps should have same dimensions", 2);
                                 }
-
+                                baseColorBitmapName = (babylonStandardMaterial.diffuseTexture != null ? Path.GetFileNameWithoutExtension(babylonStandardMaterial.diffuseTexture.name) : TextureUtilities.ColorToStringName(Color.FromArgb(255, (int)(babylonStandardMaterial.diffuse[0]), (int)(babylonStandardMaterial.diffuse[1]), (int)(babylonStandardMaterial.diffuse[2]))))
```

Comment:
This change was made to avoid situations where a generated packed texture shares a name with an unmodified texture in use in the scene. Every packed texture should have a unique name, given unique component textures/colors.